### PR TITLE
Python tweaks

### PR DESCRIPTION
**Pull request by simon-v**
A few Python tweaks and minor improvements.

### CODE REVIEW

File: python.nanorc [hunk 1]
```diff
@@ -30,23 +30,27 @@ color magenta "[.:;,+*|=!\%@]" "<" ">" "/" "-" "&"
...
 color yellow "['][^']*[^\\][']" "[']{3}.*[^\\][']{3}"
 color yellow "["][^"]*[^\\]["]" "["]{3}.*[^\\]["]{3}"
 
 ## comments
-color green "#.*$"
+color green "^#.*|[[:space:]]#.*$"
 
 ## block comments
-color green start=""""([^"]|$)" end=""""" start="'''([^']|$)" end="'''"
+#color green start=""""([^"]|$)" end=""""" start="'''([^']|$)" end="'''"
```
Comment (scopatz) [inline python.nanorc]: Please remove dead code

**Comment by scopatz** [comment]
Thanks for putting this in @simon-v!

**Comment by simon-v** [comment]
Removing commented out rules in a new commit so that it's possible to use them as reference later, if desired.

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**